src/ceph-disk

```diff
@@ -1243,14 +1245,30 @@ def update_partition(dev, description):
     group changes etc. are complete.
     """
     LOG.debug('Calling partprobe on %s device %s', description, dev)
-    command_check_call(['udevadm', 'settle'])
-    command_check_call(['partprobe', dev])
-    command_check_call(['udevadm', 'settle'])
+    partprobe_ok = False
```

Comment:
this fixes for me ceph-disk create on raw disk device under centos 7.2
I downloaded 10.0.0.3  from gitbuilder repo today, which contains that fix.
As of http://tracker.ceph.com/issues/14080 it's planned to backport to all current releases. This would be nice. Will this pull request get into 0.94.6?